runners/maven-plugin/api/maven-plugin.api

```diff
@@ -94,6 +94,11 @@ public final class org/jetbrains/dokka/maven/ExternalDocumentationLinkBuilder {
 	public final fun setUrl (Ljava/net/URL;)V
 }
 
+public class org/jetbrains/dokka/maven/HelpMojo : org/apache/maven/plugin/AbstractMojo {
+	public fun <init> ()V
+	public fun execute ()V
+}
+
```

Comment:
This addition indicates that the help task hasn't worked at most since Dokka 1.4.32 ([when the BCV was introduced](https://github.com/Kotlin/dokka/pull/1774)), and probably didn't before that.